observatory-platform/requirements.txt

```diff
@@ -1,39 +1,55 @@
+# Airflow
 apache-airflow==2.1.3
 apache-airflow-providers-slack==4.0.0
 apache-airflow-providers-http==2.0.0
```

Comment:
Nice work organising all of these in different sections :-)

Reply:
Haha thanks, I wanted to figure out what requirements could be dropped and thought it would make it easier in the future